game/scripts/npc/abilities/talents/lina_talents.txt

```diff
@@ -69,7 +69,7 @@
 			"01"
 			{
 				"var_type"					"FIELD_INTEGER"
-				"value"				"140"
+				"value"				"500" //OAA
```

Comment:
Since this is level 15 talent. 500 seems too much. 280 or 300 seems more fair.

Reply:
ok